### PR TITLE
enh(security): Using the new algorithm to hide the password in the service command line

### PR DESCRIPTION
**Pull request by callapa**
## Description

To avoid managing two different algorithms to hide the command line password of a service (https://github.com/centreon/centreon/pull/8663), we reused the Symfony container method to do so.

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x
- [x] 20.10.x (master)

## Checklist

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/include/monitoring/objectDetails/serviceDetails.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
 /*
  * Copyright 2005-2019 Centreon
```
Comment (sc979) [inline www/include/monitoring/objectDetails/serviceDetails.php:4]:
```suggestion
 * Copyright 2005-2020 Centreon
```

File: www/include/monitoring/objectDetails/serviceDetails.php [hunk 2]
```diff
@@ -66,9 +80,11 @@
...
 }
 
-if (isset($_GET["host_name"]) &&
-    $_GET["host_name"] != "" &&
-    isset($_GET["service_description"]) && $_GET["service_description"] != ""
+if (
+    isset($_GET["host_name"])
+    && $_GET["host_name"] != ""
+    && isset($_GET["service_description"])
+    && $_GET["service_description"] != ""
```
Comment (sc979) [inline www/include/monitoring/objectDetails/serviceDetails.php:87]:
```suggestion
    !empty($_GET["host_name"])
    && !empty($_GET["service_description"])
```

**Comment by UrBnW** [comment]
Hi,
Do you plan a 19.10 backport ?
Thx 👍

File: src/Centreon/Domain/HostConfiguration/HostWithServices.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Centreon\Domain\HostConfiguration;
+
+class HostWithServices
```
Comment (kduret) [inline src/Centreon/Domain/HostConfiguration/HostWithServices.php:5]: I think you can remove this class

**Comment by sc979** [comment]
@UrBnW ,
"Do you plan a 19.10 backport ?"
I don't think that the required elements are available on the 19.10.x
Regards,
sc979

File: src/Centreon/Domain/HostConfiguration/HostWithServices.php (hunk 3 above)
Comment (callapa) [reply src/Centreon/Domain/HostConfiguration/HostWithServices.php]: Yes

**Comment by UrBnW** [comment]
Finally 19.10.x backport is here : #9168.

**Pull request merged**